app/build-tools/src/teet/asset_schema.clj

```diff
@@ -131,6 +135,49 @@
 (defn- map-by-name [things]
   (into {} (map (juxt :name identity)) things))
 
+(defn- attribute-name
+  "Builds the attribute name from its `:name` and `:ctype`.
+   `(attribute-name {:name :barrier :ctype :barrierworkingwidth}) => :barrier/barrierworkingwidth`"
+  ;; TODO: Why is datatype needed here?
+  [{n :name :keys [ctype datatype] :as _attr}]
+  (when (and n ctype datatype)
+    (keyword (name ctype) (name n))))
```

Comment:
This was refactored into its own function. Is there a reason for checking for the existence of `datatype`?

Reply:
the existence checks are needed as the rotl may be incomplete and you'd get errors about entity only being used as an id